serverless.yml

```diff
@@ -201,6 +204,17 @@ stepFunctions:
         "Clean-up Cluster":
           Type: Task
           Resource: cleanup_cluster
+          Next: "Checking Drain"
+        "Checking Drain":
+          Type: Task
+          Resource: check_drain
+          Retry:
+            -
+              ErrorEquals:
+                - UndrainedInstancesException
+              IntervalSeconds: 10
+              MaxAttempts: 10
+              BackoffRate: 1
```

Comment:
Honestly not sure if this needs a catch. It's the last step, so an error should fail out anyway.